include/jail_launch_hook.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2020 Jolla Ltd.
+ * Copyright (C) 2020 Open Mobile Platform LLC.
+ * Copyright (C) 2020 Slava Monich <slava.monich@jolla.com>
```

Comment:
Also drop these.

Reply:
Drop all copyright headers or just Slava/Jolla parts?

Reply:
Drop both Jolla and Slava parts. Lower case c is preferred for (c).

Reply:
Done